src/agent/Cargo.toml

```diff
@@ -18,3 +18,11 @@ members = [
 
 [profile.release]
 lto = "thin"
+
+
+[patch.crates-io]
+# Remove after `Detegr/rust-ctrlc#81` is released.
+ctrlc = { git = "http://github.com/ranweiler/rust-ctrlc", rev = "7535c9306557ef69bc07401ba653aa5d7d625e2f" }
+
```

Comment:
can we create an issue to keep track of those two evens
- release of  `Detegr/rust-ctrlc#81`
- release of `ranweiler/pete#71`

Reply:
I was thinking that we'll already get update PRs for free from Dependabot. Do you think that's enough?

Reply:
You won't get PRs from dependabot when you've pinned to specific git hashes.

Reply:
Oh, nevermind.  That feature got added to dependabot!

`Dependabot can handle git sources, and will create PRs to keep them pointing to the latest commit`

Reply:
@bmc-msft, on top of that, this will be a good live test of what to expect, all things considered. This is a particularly interesting test case because it isn't just a git source, but a _patched_ git source!

Reply:
Yep.  More so since you control one of said repos, it means you can set a todo item for yourself to check within 24 hours of you making a commit and then release of pete.